src/modules/languages/haskell.nix

```diff
@@ -29,16 +29,25 @@ in
         Haskell language server to use.
       '';
     };
+
+    stack = lib.mkOption {
+      type = lib.types.nullOr lib.types.package;
+      default = pkgs.stack;
+      defaultText = "pkgs.stack";
+      description = ''
+        Haskell stack to use.
+      '';
+    };
   };
 
   config = lib.mkIf cfg.enable {
     packages = with pkgs; [
       cfg.package
       cfg.languageServer
```

Comment:
While working on this PR, I realized there is a bug in this line which is addressed here #645

Reply:
@domenkozar I just solved the merge conflicts caused by #645